packages/admin-ui/src/pages/wifi/components/localNetwork/LocalNetworkStatus.tsx

```
@@ -0,0 +1,111 @@
+    <p>
+      Use the Local Network Discovery service and connect to your DAppNode by
+      connecting your device to the same router as your DAppNode. Go to{" "}
+      <a href={avahiLocalDomain}>http://my.dappnode.local</a>
+      {dappnodeIdentity.internalIp === dappnodeIdentity.ip &&
+        "Local and public IPs are equal. This may be due to dappnode is running on a remote machine and does not require Local Network Discovery."}
+    </p>
+  );
+}
+
+export default function LocalNetworkStatus(): JSX.Element {
+  const avahiStatus = useApi.avahiStatusGet();
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      avahiStatus.revalidate();
+    }, 5 * 1000);
+    return () => {
+      clearInterval(interval);
+    };
+  }, [avahiStatus]);
+
+  async function enableDisableAvahi(): Promise<void> {
+    try {
+      if (avahiStatus.data) {
+        const avahiRunning = avahiStatus.data.avahiStatus === "started";
+        if (avahiRunning)
+          await new Promise<void>(resolve => {
+            confirm({
+              title: `Pause Local Network Discovery service`,
+              text: `Warning, if you are connected to your DAppNode through Local Network Discovery you may lose access to your DAppNode. Make sure to have an alternative way to connect to it, like a VPN connection.`,
+              label: "Pause",
+              onClick: resolve
+            });
+          });
+
+        await withToast(
+          () => api.avahiEnableDisable(!avahiRunning),
+
+          {
+            message: avahiRunning
+              ? "Pausing Local Network Discovery service"
+              : "Starting Local Network Discovery service",
+            onSuccess: avahiRunning
+              ? "Paused Local Network Discovery service"
+              : "Started Local Network Discovery service"
+          }
+        );
+
+        avahiStatus.revalidate();
+      }
+    } catch (e) {
+      console.error("Error on start/stop Local Network Discovery service");
```

Don't swallow errors, always log the original error at the end
```ts
console.error("your message", e)
```